.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -21,7 +21,7 @@ If you are still having issues, please be sure to include as much information as
   - OS:
   - Node.js version:
   - npm version:
-  - `dialogflow` version:
+  - `@google-cloud/dialogflow` version:
```

Comment:
So... this is a weird module.  The name actually is just `dialogflow` for historical reasons.  I am open to deprecating the old and releasing a new, but we haven't done that yet.  @bcoe as FYI

Reply:
Oh I thought all libraries default to be `@google-cloud/LibraryName` but if `dialogflow` is the module's name, I can surely change it back, thanks!

Reply:
Based on the discussion @bcoe , we will publish one last release for `dialogflow` with deprecation message and move to typescript with `@google-cloud/dialogflow`. Thanks!